### ISSUE TITLE
Droplet scRNA-seq is not zero inflated

### ISSUE THREAD

**Issue by julalipa** [vals/Blog#1]
Hi, I was wondering whether you would be willing to upload your notebook and codes for the "Droplet scRNA-seq is not zero inflated" blog entry? Hope this is not too much trouble. Thanks, Julia

**Comment by vals** [comment]
Hi Julia,

I think I wrote that post on my old work computer that was wiped when I moved, so I don't think I have it anymore.

I can remake it when I get some spare time next week (though the plots will probably look as neat).

**Comment by julalipa** [comment]
Hi Valentine, thanks a lot for getting back to me. I am mainly interested in the part of your code that fits the negative binomial model for k=0 to the dropout rates for the various datasets. I have been trying to fit this model to my own data, but I do not get such a nice result, even though my observed dropout vs mean expression plots look very similar. I am not sure if I am doing it correctly, as I am fairly new to fitting models and just wanted to compare to what you have done. I would appreciate it greatly if you re-made the plots (one is enough really), but if you are too busy don’t worry about it too much. Thanks a lot, Julia

**Comment by vals** [comment]
Hi Julia,

Ok just illustrating how I fitted the curves is pretty quick!

I made this notebook just using the Klein data (it's what I had around): https://nbviewer.jupyter.org/gist/vals/316eeaf9b8c796f8c32037488c1519a6

I just find the phi in the NB distribution using a curve fitting functions on the mean-variance relation. Then I use this value in the theoretical probability of zero counts.

I hope the code is self explanatory, but just let me know if something is unclear.

/Valentine
